hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -3203,10 +3206,90 @@ func reconcileMachineApproverDeployment(deployment *appsv1.Deployment, hc *hyper
 	return nil
 }
 
-const oidcDocumentsFinalizer = "hypershift.io/aws-oidc-discovery"
+const (
+	oidcDocumentsFinalizer         = "hypershift.io/aws-oidc-discovery"
+	serviceAccountSigningKeySecret = "sa-signing-key"
+	serviceSignerPublicKey         = "service-account.pub"
+	jwksURI                        = "/openid/v1/jwks"
+	discoveryTemplate              = `{
```

Comment:
Just curious, is there some Go type for this I'm missing we could be using instead of templating?

Reply:
None of which I'm aware